synapseclient/client.py

```diff
@@ -189,7 +189,8 @@ class Synapse(object):
 
     # TODO: add additional boolean for write to disk?
     def __init__(self, repoEndpoint=None, authEndpoint=None, fileHandleEndpoint=None, portalEndpoint=None,
-                 debug=None, skip_checks=False, configPath=CONFIG_FILE, requests_session=None, cache_root_dir=None):
+                 debug=None, skip_checks=False, configPath=CONFIG_FILE, requests_session=None,
+                 silent=None, cache_root_dir=None):
```

Comment:
It would be safer to add the new optional silent arg as the last argument, that way if anyone was passing positional for all these values it would still assign the value to the proper variable (unlikely but possible).